code/modules/projectiles/projectile/beams.dm

```diff
@@ -909,10 +909,22 @@ var/list/laser_tag_vests = list(/obj/item/clothing/suit/tag/redtag, /obj/item/cl
 		for(var/datum/reagent/R in reagents.reagent_list)
 			reagents.add_reagent(R.id, reagents.get_reagent_amount(R.id))//so here we're just doubling our quantity of reagents from 10 to 20
 		if(istype(A, /mob))
-			var/splash_verb = pick("douses","completely soaks","drenches","splashes")
-			A.visible_message("<span class='warning'>\The [src] [splash_verb] [A]!</span>",
-								"<span class='warning'>\The [src] [splash_verb] you!</span>")
-			splash_sub(reagents, get_turf(A), reagents.total_volume/2)//then we splash 10 of those on the turf in front (or under in case of mobs) of the hit atom
+			if(def_zone == LIMB_HEAD && ishuman(A) && prob(20)) //if aiming at head and is humanoid
```

Comment:
You actually CAN have this be if they're targeting the mouth.

Do `firer_mob.zone_sel.selecting == TARGET_MOUTH`

Now, this will only check where they were aiming, not where they actually hit (which is your problem). You could combine it with the existing check though -- if they hit the head, and they were aiming for the mouth, they get the chance to squirt inside. That seems fair to me, you could even up the prob I think, because as it is currently you'll sometimes squirt in their mouth even if you were aiming at their chest, etc.

Reply:
Done, decided to remove probability checking altogether since that was a backup for not being able to mouth aim